README.md

```diff
@@ -1 +1,181 @@
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Install](#install)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ (OPTIONAL) ](#-faq-optional-)
+  - [📝 License ](#-license-)
+  
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Awesome Books] <a name="about-project"></a>
+
+- [Awesome books](#-awesome-books-)  is a simple website that displays a list of books and allows you to add and remove books from that list.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+     <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+     <li><a href="https://www.w3schools.com/css/">Javascript</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- N/A
+  
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Google Chrome (or any browser)
+ - Git
+ - Visual studio code (or other code editors)
+  
+### Setup
+
+Clone this repository to your desired folder:
+> Opend GitBash in your computer and type this (replace 'my-folder' with the name of your desired folder)
+sh
+  cd my-folder
+sh
+  git clone git@github.com:Pi1998/first-capstone.git
+
+### Usage
+
+To run the project, execute the following command:
+sh
+  code .
+
+### Install
+
+> You don't need to install anything in your computer since it is a webpage. Instead, you can clone this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 Shinn Thant Swam Ye
+
+- GitHub: [@Pi1998](https://github.com/Pi1998)
+- LinkedIn: [Shinn Thant Swam Ye](https://www.linkedin.com/in/shinn-thant-swam-ye-617a66253/)
+  
+  
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] [CSS upgrades]
+- [ ] [JavaScript Functions]
+- [ ] [Accessibility Features]
+- [ ] [Mobile and desktop responsiveness]
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+- [Issues](https://github.com/Pi1998/awesome-books)
```

Comment:
Great job team🤝

- [ ] [Optional] Kindly ensure the issues link redirects to the correct page. It is supposed to link the navigator to the issues page not the repository.